docs/src/utils/codeExampleUtils.ts

```diff
@@ -6,12 +6,15 @@ export function trimMainBlock(lines: string[]): string[] {
   return mainIndex !== -1 ? lines.slice(0, mainIndex) : lines;
 }
 
+const IGNORED_COMMENT_TYPES = ['noqa', 'type: ignore'];
```

Comment:
Could `isort: skip_file` be added here? https://linear.app/dagster-labs/issue/DOC-959/filter-isort-comments-in-codeexample-component